test/spec/adapters/sekindoUM_spec.js

```diff
@@ -0,0 +1,88 @@
+import {expect} from 'chai';
+import _ from 'lodash';
```

Comment:
This import doesn't seem to be used anywhere, and lodash isn't listed in `package.json` as a dependency. This line could be removed if it isn't needed